src/components/ResetPassword/index.jsx

```diff
@@ -0,0 +1,95 @@
+/* eslint-disable react/prop-types */
+import React, { useState, useRef } from 'react';
+import { connect } from 'react-redux';
+import { ClipLoader } from 'react-spinners';
+import Modal from '../../views/Modal.js';
+import passwordResetRequest from '../../actions/resetPassword';
+
+
+/**
+ * Form for resetting password
+ * @param {object} props Component props
+ * @param {function} props.passwordResetRequest action creator function
+ */
+function ResetPassword(props) {
+  const [userEmail, setUserEmail] = useState({
+    email: null,
+  });
+
+  const { isLoading } = props;
+
+  const [message, setMessage] = useState(null);
+  const input = useRef(null);
+  
+  const updateInput = event => {
+    setUserEmail({ ...userEmail, [event.target.name]: event.target.value });
+  };
+
+  const clearInputs = () => {
+    input.current.value = null;
+    setUserEmail({ ...userEmail, email: null });
+  };
+
+  const handleSubmit = () => {
+    setMessage(null);
+    props.passwordResetRequest(userEmail.email).then(res => {
```

Comment:
#### Nitpick
- This is anti-pattern with regards to flux architecture. Subscribe to whatever it is you need in the `redux store` and use that to handle this.

Reply:
okay! this has been taken care of now.